### PR TITLE
Refactor and fix minor mistakes

### PR DESCRIPTION
**Pull request by Jerry-py**
## Description

1. Fix codespell check
2. Refactor code
3. Update discord api version to v10

## Checklist

- [ ] - All code changes have been tested, if any
- [ ] - The documentation has been updated accordingly for any code changes
- [ ] - This PR fixes an already existing issue
- [ ] - This PR adds a new feature (new functions, methods, classes, parameters, etc.)
- [ ] - This PR fixes a bug
- [x] - This PR is a breaking change (removes/renames functions, parameters etc.)
- [ ] - This PR is NOT a code change (changes Documentation files, README.md, CONTRIBUTING.md etc.)

### CODE REVIEW

**Review by nav-github01001 — CHANGES_REQUESTED** [review]
I have fixed the opus issues. you can delete dir #73

**Comment by Jerry-py** [comment]
K

**Review by Jerry-py — COMMENTED** [review]
Done

**Review by nav-github01001 — APPROVED** [review]
Looks Good

File: EpikCord/__init__.py [hunk 1]
```diff
@@ -887,8 +885,8 @@ def __init__(self, client, data: dict):
     async def delete(self, *, reason: Optional[str] = None) -> None:
         if reason:
             headers = self.client.http.headers.copy()
-            if reason:
-                headers["reason"] = reason
```
Comment (TheUntraceable) [inline EpikCord/__init__.py:891]: Need to fix this later

File: .github/workflows/codespell.yml [hunk 2]
```diff
@@ -7,4 +7,4 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-python@v2
       - run: pip install codespell
-      - run: codespell
\ No newline at end of file
+      - run: codespell --ignore-words-list="somwhere"
```
Comment (TheUntraceable) [inline .github/workflows/codespell.yml:10]: That's spelt wrong?

File: EpikCord/__init__.py [hunk 3]
```diff
@@ -434,7 +434,7 @@ def event(self, func):
 
     async def ready(self, data: dict):
         self.user: ClientUser = ClientUser(self.session, data.get("user"))
-        application_response = await self.session.get("https://discord.com/api/v9/oauth2/applications/@me", headers={"Authorization": f"Bot {self.token}"})
+        application_response = await self.session.get("https://discord.com/api/v10/oauth2/applications/@me", headers={"Authorization": f"Bot {self.token}"})
```
Comment (TheUntraceable) [inline EpikCord/__init__.py:437]: image
V10 isn't documented *yet*, I'd like to add it once it's the recommended one version to use.

Comment (Jerry-py) [reply EpikCord/__init__.py:513]: K

**Comment by Jerry-py** [comment]
v10 is not stable

**Pull request closed without merging**